tests/e2e/specs/out-of-game/localization.spec.js

```diff
@@ -72,3 +72,50 @@ describe('Localization', () => {
     checkLobbyTranslation(fr);
   });
 });
+
+describe('language files', () => {
+  function extractKeys(obj, keyList = []) {
+    Object.keys(obj).forEach(key => {
+      if (typeof obj[key] === 'object' && obj[key] !== null) {
+        extractKeys(obj[key], keyList);
+      } else {
+        keyList.push(key);
+      }
+    });
+    return keyList;
+  }
+
+  let enKeys = extractKeys(en).sort();
+  let frKeys = extractKeys(fr).sort();
+  let esKeys = extractKeys(es).sort();
+
+  it('English should have no empty strings', () => {
+    enKeys.forEach((key) => {
+      expect(en[key]).to.not.eql('');
+    });
```

Comment:
We can spruce up the assertion to be more specific and legible in case of failures, using the 2nd argument of `.eq()` to write an assertion message. Also we generally use the `.eq()` assertion rather than `.eql()`. There's no difference, but it's best to have a singular convention.

```suggestion
    enKeys.forEach((key) => {
      const assertionMsg = `en.json value at key ${key} should be non-empty, but it was empty`;
      expect(en[key]).to.not.eq('', assertionMsg);
    });
```